indico/web/client/js/react/components/notes/NoteConflictModal.jsx

```diff
@@ -0,0 +1,190 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import moment from 'moment/moment';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {
+  Card,
+  CardContent,
+  Header,
+  Modal,
+  Button,
+  Message,
+  Icon,
+  Grid,
+  GridRow,
+  GridColumn,
+  List,
+  Segment,
+} from 'semantic-ui-react';
+
+import {Param, Translate} from 'indico/react/i18n';
+import {camelizeKeys} from 'indico/utils/case';
+
+import './NoteConflictModal.module.scss';
+
+export function NoteConflictModal({
+  currentNote,
+  externalNote,
+  onClose,
+  setCloseAndReload,
+  overwriteChanges,
+}) {
+  const [open, setOpen] = useState(true);
+
+  // camelize the externalNoteSource keys
+  const extNote = camelizeKeys(externalNote);
+
+  const renderDeletedRevision = () => {
+    return (
+      <Segment placeholder>
+        <Header icon>
+          <Icon name="trash outline alternate" />
+          <Translate>This note revision has been deleted.</Translate>
+        </Header>
+      </Segment>
+    );
+  };
+
+  return (
+    <Modal
+      open={open}
+      onClose={() => {
+        setOpen(false);
+        onClose();
+      }}
+      onOpen={() => setOpen(true)}
+      size="large"
```

Comment:
do we need to specify the size if we're already setting a `min-width`?

Reply:
Just keeping `size="large"` also works, so I'll drop the `min-width`